src/batch-client/index.ts

```diff
@@ -474,6 +477,16 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		}).then(res => mapValuesDeep(res._attrs, coerceStringToBoolean));
 
+	public recoverAccount = ({ email, op }: RecoverAccountOptions) =>
+		this.jsonRequest({
+			name: 'RecoverAccount',
+			body: {
+				channel: SetRecoveryAccountChannelType.email,
```

Comment:
we shouldn't be hardcoding channel here, we can specify it as default parameter in query